source/docs/steeltoe-circuitbreaker.html.markdown

```diff
@@ -5,53 +5,53 @@ date: 2016/4/1
-
-In the future you can expect to see more Circuit Breaker options as part of Steeltoe.
+There are several options to choose from when implementing the Circuit Breaker pattern. Steeltoe has initially chosen to support one based on Hystrix, Netflix's Latency and Fault Tolerance library for distributed systems. For more information about Hystrix, see the [Netflix/Hystrix Wiki](https://github.com/Netflix/Hystrix/wiki) and the [Spring Cloud Netflix](http://projects.spring.io/spring-cloud/) documentation.
 
 # 0.0 Initialize Dev Environment
 
-All of the Steeltoe sample applications are in the same repository. If you haven't already, use git to clone the repository or download with your browser from GitHub: [Steeltoe Samples](https://github.com/SteeltoeOSS/Samples)
+All of the Steeltoe sample applications are in the same repository. If you have not already done so, use Git to clone the repository, as follows:
 
 ```bash
 > git clone https://github.com/SteeltoeOSS/Samples.git
 ```
 
-> Note: all Circuit Breaker samples in the Samples repository have a base path of `Samples/CircuitBreaker/src/`
+Alternatively, you can download it with your browser from GitHub at [Steeltoe Samples](https://github.com/SteeltoeOSS/Samples).
+
+> Note: all Circuit Breaker samples in the Samples repository have a base path of `Samples/CircuitBreaker/src/`.
 
-Make sure your Cloud Foundry CLI tools are logged in and targeting the correct org and space:
+Make sure your Cloud Foundry CLI tools are logged in and targeting the correct org and space, as follows:
 
 ```bash
 > cf login [-a API_URL] [-u USERNAME] [-p PASSWORD] [-o ORG] [-s SPACE] [--skip-ssl-validation]
 or
 > cf target -o <YourOrg> -s <YourSpace>
 ```
 
-The CircuitBreaker sample requires a Eureka server and a Hystrix dashboard. We'll cover how to run those later, but if you intend to run the samples locally, install the Java 8 JDK and Maven 3.x now.
+The CircuitBreaker sample requires a Eureka server and a Hystrix dashboard. We cover how to run those later. If you intend to run the samples locally, install the [Java 8 JDK](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html) and [Maven 3.x](https://maven.apache.org/download.cgi) now.
 
 # 1.0 Netflix Hystrix
 
-The Steeltoe Hystrix framework enables application developers to isolate and manage back-end dependencies so that a single failing dependency does not take down the entire application. This is accomplished by wrapping all calls to external dependencies in a `HystrixCommand` which is able to execute in its own separate external thread.
+The Steeltoe Hystrix framework lets application developers isolate and manage back-end dependencies so that a single failing dependency does not take down the entire application. This is accomplished by wrapping all calls to external dependencies in a `HystrixCommand`, which runs in its own separate external thread.
```

Comment:
All references to "Steeltoe Hystrix" is probably not allowed as a possibly trademark infraction with Netflix Hystrix.  I would recommend we change all "Steeltoe Hystrix" to "Steeltoe's Hystrix implementation"